connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/CheckConnectJvmClientCompatibility.scala

```diff
@@ -145,6 +145,7 @@ object CheckConnectJvmClientCompatibility {
 
       // DataFrameNaFunctions
       ProblemFilters.exclude[Problem]("org.apache.spark.sql.DataFrameNaFunctions.this"),
+      ProblemFilters.exclude[Problem]("org.apache.spark.sql.DataFrameNaFunctions.fillValue"),
```

Comment:
@LuciferYang  it seems the MiMa tests also check `private[sql]`, is this as expected?

Reply:
This behavior is expected now, but I think some refactoring work is needed to automatically filter similar checks, just like what `dev/mima` check does.

Reply:
got it, thanks

Reply:
filed https://issues.apache.org/jira/browse/SPARK-43246 to tracking this